addons/account/demo/account_demo.py

```diff
@@ -552,6 +739,10 @@ def _get_demo_account(self, xml_id, account_type, company):
                 ('model', '=', 'account.account'),
                 ('module', '=like', 'l10n%')
```

Comment:
You can just remove this line, the accounts are always tied to `account` anyway since 16.2